### PR TITLE
chore: fix versions to align with last beta release

### PR DESCRIPTION
**Pull request by yuth**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by lazpavel — APPROVED** [review]

**Review by cjihrig — APPROVED** [review]

**Review by ammarkarachi — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -16801,6 +16801,11 @@ libnpmpublish@^4.0.0:
     semver "^7.1.3"
     ssri "^8.0.1"
 
+libphonenumber-js@1.9.47:
```
Comment (jhockett) [inline yarn.lock:16804]: This is a fairly large dependency (7.11Mb unpacked according to [npm](https://www.npmjs.com/package/libphonenumber-js), can it be deduped with `^1.7.31`?

File: yarn.lock [hunk 2]
```diff
@@ -23651,11 +23656,16 @@ typescript@^3.8.3, typescript@^3.9.3:
   resolved "https://registry.npmjs.org/typescript/-/typescript-3.9.10.tgz#70f3910ac7a51ed6bef79da7800690b19bf778b8"
   integrity sha512-w6fIxVE/H1PkLKcCPsFqKE7Kv7QUwhU8qQY2MueZXWx5cPZdwFupLgKK3vntcK98BtNHZtAF4LA/yl2a7k8R6Q==
 
-typescript@^4.1.3, typescript@^4.4.3:
+typescript@^4.4.3:
   version "4.5.4"
   resolved "https://registry.npmjs.org/typescript/-/typescript-4.5.4.tgz#a17d3a0263bf5c8723b9c52f43c5084edf13c2e8"
   integrity sha512-VgYs2A2QIRuGphtzFV7aQJduJ2gyfTljngLzjpfW9FoYZF6xuw1W0vW9ghCKLfcWrCFxK81CSGRAvS1pn4fIUg==
 
+typescript@^4.5.5:
```
Comment (jhockett) [inline yarn.lock:23664]: Same for [Typescript](https://www.npmjs.com/package/typescript), 64Mb unpacked

Comment (yuth) [reply yarn.lock:23664]: This should not be bundled as this is `devDependency`. I updated it everywhere except for the resources

File: yarn.lock (hunk 1 above)
Comment (yuth) [reply yarn.lock:16804]: I had to pin this to latest version which builds. There has been an update to this package and they had had a breaking change which fails compilation

**Review by jhockett — APPROVED** [review]

**Pull request merged**